README.md

```diff
@@ -1,3 +1,3 @@
-# ros2-project-template
+# jetson\_camera\_trigger
```

Comment:
backslash is necesarry?

Reply:
@wep21 I appreciate your suggestion. As the backslashes you pointed out are surely unnecessary, I made a commit to remove them.